composer.json

```diff
@@ -14,13 +14,18 @@
         }
     ],
     "require": {
-        "php": ">=5.4.0",
-        "symfony/process": ">=2.2"
+        "php": "^7.1",
+        "doctrine/instantiator": "1.0.*",
+        "phpdocumentor/reflection-docblock": "2.0.*",
+        "sebastian/comparator": "^3.0",
+        "sebastian/recursion-context": "^3.0",
```

Comment:
They were introduced upstream and came in with those pulls.
Yet interestingly if I just delete them completely and run composer update the tests all still pass.

Will do some snooping around to learn more.

Reply:
From what I found the upstream added a commit to "force dependency version for CI check".
I think as long as tests pass on all PHP versions this may not be a concern for our fork here. Will test.